drivers/elum_carlogavazzi_em210.json

```diff
@@ -0,0 +1,31 @@
+{
+  "common": {
+    "fncode": 3,
+    "words": 2,
+    "datatype": "float",
+    "order": "lsr"
+  },
+  "fields": {
+    "V_L1": {"register":  56, "unit": "V", "description": "Phase 1 line to neutral volts"},
```

Comment:
super minot, but a lot of these have a double space after `"register":`. Not sure if intentional?

Reply:
Removed extra white space